changelog/february2024/2024-02-20-object-storage-added-object-storage-introduces-bucket-u.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Object Storage introduces bucket use case to customize your experience
+status: added
+author:
+    fullname: 'Join the #object-storage channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2024-02-20
+category: storage
+product: object-storage
+---
+
+Bucket use case can be filled up at bucket creation into the Scaleway Console or later on edited into Bucket settings. We encourage to filled in bucket use case in order to benefit from customized recommendations, while we work for enriching even more your experience.
```

Comment:
```suggestion
At the time of creating a bucket in the Scaleway Console, you have the option to fill in the bucket use case. Optionally, you can edit this information in the Bucket settings later on. We recommend filling in the bucket use case to unlock personalized recommendations tailored to your needs. We are continuously enhancing our services to further enrich your experience.
```

Reply:
What about 

"Upon creation of a bucket on the Scaleway console, you can fill in its use case. Optionally, you can edit this information in the Bucket settings later on. We recommend filling in the bucket use case to unlock personalized recommendations tailored to your needs. We are continuously enhancing our services to further enrich your experience." ?